### PR TITLE
Asset versioning, code cleaning, comments, style changes

### PR DESCRIPTION
**Pull request by christianbator**

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -40,7 +41,7 @@ def handle_500(error):
 @app.route("/post_text", methods=["POST"])
 def handle_post_text():
     # Authenticate
-    (sitename, authenticate_error) = authenticate_post_request(request)
+    (host, authenticate_error) = authenticate_post_request(request)
```
Comment (christianbator) [inline app/app.py:44]: Renamed "sitename" to "host"

File: app/markdown_parser.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ def parse_caption_markdown(markdown):
     parsed_text = re.sub(r"_(.+)_", lambda x: "<i>" + x.group(1) + "</i>", parsed_text)
 
     # Replace urls
-    parsed_text = re.sub(r"\[(.+?)\]\((.+?)\)", lambda x: "<a target='_blank' rel='noopener' href=\"" + x.group(2) + "\">" + x.group(1) + "</a>", parsed_text)
+    parsed_text = re.sub(r"\[(.+?)\]\+\((.+?)\)", lambda x: f"<a target='_blank' rel='noopener' href=\"" + x.group(2) + "\">" + x.group(1) + "</a>", parsed_text)
```
Comment (christianbator) [inline app/markdown_parser.py:27]: So whatcha think about doing `[text]+(url)` to open in a new tab?
and `[text](url)` to open in current window?

Comment (benmathia) [reply app/markdown_parser.py:27]: I'm not undown

File: template/templates/index.html [hunk 3]
```diff
@@ -1,23 +1,42 @@
...
-    <link rel="icon" href="../icons/favicon.png">
-    <link rel="shortcut icon" href="../icons/favicon.png">
-    <link rel="apple-touch-icon" href="../icons/apple-touch-icon.png">
-    <link rel="apple-touch-icon-precomposed" href="../icons/apple-touch-icon.png">
   </head>
   <body role="main">
-    <p class="title"><a href="/">{title}</a></p>
-    <p class="subtitle">{description}</p>
-{posts}
+<!-- Header -->
```
Comment (benmathia) [inline template/templates/index.html]: Needs a lil indent there

File: template/templates/post.html [hunk 4]
```diff
@@ -1,25 +1,40 @@
...
+    {post}
+
+    <!-- Footer -->
     <div>
-        <img class="footer_image" src="../icons/favicon.png">
+        <img class="footer_image" src="../icons/favicon-{icon_version}.png">
     </div>
+
+    <!-- Video Control Script -->
     <script>
```
Comment (benmathia) [inline template/templates/post.html]: 😞

File: app/content_manager.py [hunk 5]
```diff
@@ -106,27 +125,50 @@ def create_video_post(self, video_path, caption, location):
...
+
+        self.add_post_and_update(post = post)
 
-        # Add post
-        self.add_post_and_update(post)
         return post
 
+    #
+    # Utilities
+    #
```
Comment (benmathia) [inline app/content_manager.py:146]: Gonna need a newline after this here `#`

File: app/properties.py [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+# properties.py
+#
+
+import json
+
+class Properties:
+
+    def __init__(self, json):
+        self.site_title = json["site_title"]
+        self.display_title = json["display_title"]
```
Comment (benmathia) [inline app/properties.py:11]: What's the difference between `site_title` and `display_title`?

File: app/post.py [hunk 7]
```diff
@@ -57,75 +50,81 @@ def __str__(self):
 #
 
 class TextPost(Post):
-    def __init__(self, post_id, text_posts_dir, text_filename, date, location):
+    def __init__(self, post_id, text_posts_dir, text_filename, location, date):
```
Comment (benmathia) [inline app/post.py:54]: We doing newlines between the class declaration and body?

**Comment by benmathia** [comment]
I dig it, haven't tested it on my site yet, though. Probably gotta migrate my `properties.json`, but I like the idea of moving some more stuff into there.

File: app/properties.py (hunk 6 above)
Comment (christianbator) [reply app/properties.py:11]: `site_title` is what the browser shows and `display_title` is what is shown on the page. Most of the time I imagine they're the same, but perhaps someone wants `Ben's Blog` to appear on the tab, but `flippin bits` to be the actual blog name

Comment (christianbator) [reply app/properties.py:11]: We don't need it - mostly a carry over from my site issues :)

Comment (benmathia) [reply app/properties.py:11]: I dig it

**Pull request merged**